### PR TITLE
Week10

### PR DESCRIPTION
**Pull request by dpshelio**
Last bits

I still need to add the profiler bit.

### CODE REVIEW

File: ch07dry/049Operators.ipynb [hunk 1]
```diff
@@ -0,0 +1,533 @@
+{
```
Comment (dpshelio) [inline ch07dry/049Operators.ipynb]: @ageorgou  - please, review this *new* notebook. I followed your sugestions.

File: ch08performance/040cython.ipynb [hunk 2]
```diff
@@ -5,7 +5,7 @@
    "metadata": {},
    "source": [
     "## Cython\n",
-    "Cython can be viewed as an extension of Python where variables and functions are annotated with extra information, in particular types. The resulting Cython source code will be compiled into optimized C or C++ code, and thereby yielding substantial speed-up of slow Python code. In other word, cython provides a way of writting Python with  comparable performance to that of C/C++."
+    "Cython can be viewed as an extension of Python where variables and functions are annotated with extra information, in particular types. The resulting Cython source code will be compiled into optimized C or C++ code, and thereby yielding substantial speed-up of slow Python code. In other words, Cython provides a way of writting Python with  comparable performance to that of C/C++."
```
Comment (ageorgou) [inline ch08performance/040cython.ipynb]:
```suggestion
    "Cython can be viewed as an extension of Python where variables and functions are annotated with extra information, in particular types. The resulting Cython source code will be compiled into optimized C or C++ code, and thereby yielding substantial speed-up of slow Python code. In other words, Cython provides a way of writing Python with comparable performance to that of C/C++."
```

File: ch08performance/020numpy.ipynb [hunk 3]
```diff
@@ -1841,14 +1853,147 @@
...
+   "metadata": {},
+   "source": [
+    "Still slower. Probably due to lots of copies -- the point here is that you need to *experiment* to see which optimisations will work. Performance programming needs to be empirical."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Profiling"
```
Comment (dpshelio) [inline ch08performance/020numpy.ipynb]: @ageorgou - new section for you to review :stuck_out_tongue_closed_eyes:

**Comment by ageorgou** [comment]
I had comments :(

**Pull request merged**